gcis.ttl

```diff
@@ -1032,4 +1032,10 @@ gcis:hasSubOrganization  a owl:ObjectProperty ;
 gcis:isSupportedBy  a owl:ObjectProperty ;
 	rdfs:label "Is Supported By" ;
 	rdfs:comment "A finding is supported by data, models, model runs, software, etc.";
-	rdfs:domain gcis:Finding .	
+	rdfs:domain gcis:Finding .
+
+gcis:descriptionOfOfOutputFiles a owl:DatatypeProperty ;
+	rdfs:label "description of output files" ;
```

Comment:
and semicolon in-lieu of a colon on line 1039:
rdfs;comment "narrative description of one or more files generated by an
activity" ;

On Wed, Jun 17, 2015 at 3:15 PM, Brian Duggan <notifications@github.com>
wrote:

> In gcis.ttl
> <https://github.com/USGCRP/gcis-ontology/pull/20#discussion_r32663752>:
>
> > @@ -1032,4 +1032,10 @@ gcis:hasSubOrganization  a owl:ObjectProperty ;
> >  gcis:isSupportedBy  a owl:ObjectProperty ;
> >  	rdfs:label "Is Supported By" ;
> >  	rdfs:comment "A finding is supported by data, models, model runs, software, etc.";
> > -	rdfs:domain gcis:Finding .	
> > +	rdfs:domain gcis:Finding .
> > +
> > +gcis:descriptionOfOfOutputFiles a owl:DatatypeProperty ;
> > +	rdfs:label "description of output files" ;
>
> little typo: OfOf -> Of
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGCRP/gcis-ontology/pull/20/files#r32663752>.
>



-- 
_________________________________
Justin Goldstein, Ph.D.
Advance Science Climate Data and Observing Systems Coordinator
US Global Change Research Program
1717 Pennsylvania Ave NW, Suite #250
Washington, DC 20006

O: (202) 419-3496
M: (202) 285-3005

e-mail: jgoldstein AT usgcrp Dot gov
http://www.globalchange.gov